server/src/event/event.service.ts

```diff
@@ -44,6 +44,13 @@ export class EventService {
     return true;
   }
 
+  /** Retrieves default event */
+  async getDefaultEvent() {
+    return await this.prisma.eventBase.findUniqueOrThrow({
+      where: { isDefault: true },
+    });
+  }
+
```

Comment:
We have just one default event for now, but in the future the event retrieved should depend on the organization (assuming diff orgs have diff default events?)

Reply:
Actually this method isn't used anywhere, should I just delete it